### ISSUE TITLE
ARIA: Lint Failures

### ISSUE THREAD

**Issue by melloware** [primefaces/primereact#2819]
### Describe the bug

- [x] ./components/lib/autocomplete/AutoComplete.js

498:21  Warning: Elements with the ARIA role "combobox" must have the following attributes defined: aria-controls,aria-expanded  jsx-a11y/role-has-required-aria-props

- [ ] ./components/lib/multistatecheckbox/MultiStateCheckbox.js

124:21  Warning: Elements with the ARIA role "checkbox" must have the following attributes defined: aria-checked  jsx-a11y/role-has-required-aria-props

- [ ] ./components/lib/tristatecheckbox/TriStateCheckbox.js

73:21  Warning: Elements with the ARIA role "checkbox" must have the following attributes defined: aria-checked  jsx-a11y/role-has-required-aria-props

### Reproducer

Just run `npm run lint`

### PrimeReact version

8.0.1

### React version

18.x

### Language

ALL

### Build / Runtime

Next.js

### Browser(s)

ALL

### Steps to reproduce the behavior

Just run `npm run lint`

### Expected behavior

No lint failures
